### ISSUE TITLE
Publish to bower

### ISSUE THREAD

**Issue by fernandoluizao** [jquery-form/form#17]
Hi,

Thanks for the great effort of maintaining this plugin! Will it be published to bower?

**Comment by kevindb** [comment]
Unfortunately, the canonical name `jquery-form` is already reserved in Bower for the upstream repo at [malsup/form](https://github.com/malsup/form). The most efficient solution would be for @malsup to reassign ownership of the upstream repo to the 'jquery-form' organization on Github.

**Comment by fernandoluizao** [comment]
Seems like he's inactive on github. Is it worth trying to contact by email?

**Comment by kevindb** [comment]
I at-messaged him on Twitter and sent him an email, with no response. You should feel free to try contacting him as well. I don't want to pester him unnecessarily, but contact from another person would show community support.

**Comment by malsup** [comment]
Kevin, I was looking at this yesterday.  It appears that I can't transfer ownership if your account has a repo named "form".

>The target account must not have a repository with the same name, or a fork in the same network.

From:  https://help.github.com/articles/about-repository-transfers/

**Comment by kevindb** [comment]
@malsup You're right. I've invited you to be an owner of the org (also a requirement). The requirements also say "must not have...a fork in the same network".

If you're ok transferring your repo, I'll delete this existing repo to 'make room' for the transfer.

**Comment by malsup** [comment]
Sure, I'll transfer it.

**Comment by kevindb** [comment]
Thank you so much, @malsup! I'll delete this repo from Github after sending this message.
